### PR TITLE
controller:lib:add support for vendor specific JDKS IPV4 CONTROL

### PR DESCRIPTION
**Pull request by vronchetti**

### CODE REVIEW

File: controller/lib/src/control_descriptor_imp.cpp [hunk 1]
```diff
@@ -46,5 +49,90 @@ control_descriptor_response * STDCALL control_descriptor_imp::get_control_respon
...
+        log_imp_ref->post_log_msg(LOGGING_LEVEL_ERROR, "aem_cmd_get_control_resp_read error\n");
+        assert(aem_cmd_get_control_resp_returned >= 0);
+        return -1;
+    }
+
+    store_cmd_resp_frame(AEM_CMD_GET_CONTROL, frame, ETHER_HDR_SIZE, frame_len);
+
+    msg_type = control_resp.aem_header.aecpdu_header.header.message_type;
+    status = control_resp.aem_header.aecpdu_header.header.status;
+    u_field = control_resp.aem_header.command_type >> 15 & 0x01; // u_field = the msb of the uint16_t command_type
```
Comment (andrew-elder) [inline controller/lib/src/control_descriptor_imp.cpp:132]: This is unsolicited flag?

**Comment by andrew-elder** [comment]
Build is failing for some reason?

File: controller/lib/src/control_descriptor_imp.cpp (hunk 1 above)
Comment (vronchetti) [reply controller/lib/src/control_descriptor_imp.cpp:132]: yes

**Comment by vronchetti** [comment]
> Build is failing for some reason?

looks like missing doxygen..

**Comment by vronchetti** [comment]
> Build is failing for some reason?

missing jdk submodule update - fixed now, closing

**Pull request merged**